torch_xla/distributed/xla_backend.py

```diff
@@ -42,13 +41,6 @@ def __init__(self, prefix_store, rank, size, timeout):
     self.prefix_store = prefix_store  # reserved for future use.
     self.timeout = timeout
     self._mesh = []
-    # Initialize xrt neuron environment
-    # Passes in the store created by torch.distributed to avoid
-    # creating two TCP stores. We only want to call this
-    # when the user is using torchrun and not xmp.spawn()
-    # or some other flow.
-    if os.getenv('TORCHELASTIC_RUN_ID') != None:
```

Comment:
Note, we should give AWS folks a heads up when we merge this pr.

Reply:
got it, have added in the list of full list of OpenXLA migration